README.md

```diff
@@ -2,6 +2,7 @@
 
 This is the code base for the UTK EF interdisciplinary design course application site.
 This shouldn't get pushed. 
+This really shouldn't get pushed.
```

Comment:
Would you delete these test lines from the README, where we were testing the repo config?

Reply:
Sure